### PR TITLE
About page

### PR DESCRIPTION
**Pull request by meeragk99**

### CODE REVIEW

**Review by xchanti124 — COMMENTED** [review]
Looks good to me, however, I think you should wait for Olya's approval.

File: package.json [hunk 1]
```diff
@@ -7,14 +7,15 @@
     "@testing-library/jest-dom": "^5.16.5",
     "@testing-library/react": "^13.4.0",
     "@testing-library/user-event": "^13.5.0",
+    "express-body-parser-error-handler": "^1.0.4",
```
Comment (FlyingRatMan) [inline package.json]: We don't need this "express-body-parser-error-handler". Could you please delete it? `npm uninstall express-body-parser-error-handler`

File: package.json [hunk 2]
```diff
@@ -7,14 +7,15 @@
...
     "google-map-react": "^2.2.0",
     "jwt-decode": "^3.1.2",
-    "react-dotenv": "^0.1.3",
-    "express-body-parser-error-handler": "^1.0.4",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
+    "react-hook-form": "^7.42.1",
```
Comment (FlyingRatMan) [inline package.json]: We don't need this "react-hook-form". Could you please delete it? `npm uninstall react-hook-form`

File: src/App.js [hunk 3]
```diff
@@ -1,29 +1,28 @@
...
-          <Route path={"details"} element={<Location />} />
-          <Route path={"location_form"} element={<LocationForm />} />
-        </Route>
-        {/* <Route path={"/new_location"} element={<LocationForm/>} /> */}
+        <Route path={"/"} element={<LocationsList />} />
+        <Route path={"/details/:id"} element={<Location />} />
+        <Route path={"/location_form"} element={<LocationForm />} />
+        <Route path={"/about"} element={<About />} />
+        <Route path={"/login"} element={<div>Login page</div>} />
+        <Route path={"/404"} element={<Error404 />} />
```
Comment (FlyingRatMan) [inline src/App.js]: Are we redirecting explicitly to `/404` anywhere? We already have a fallback for undefined routes.

File: src/components/About.js [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+      <h1>About Us</h1>
+      <p>We are a team of blooming developers building our first project under TechLabs.</p>
+      <h2>Our Mission</h2>
+      <p>
+        Our mission is to create a platform where folks can share their discovered locales that are close to their
+        hearts with the rest of the world.
+      </p>
+      <p>Happy Discovering!!</p>
+      <p>-Chantal,Olya,Dominik and Meera.</p>
+      <img src="/path/to/image" alt="Image description" />
```
Comment (FlyingRatMan) [inline src/components/About.js]: This should be removed.

File: src/components/About.js [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+      <button aria-label="Click me">Click me</button>
```
Comment (FlyingRatMan) [inline src/components/About.js]: This should be removed.

**Review by FlyingRatMan — CHANGES_REQUESTED** [review]
There are things to be removed.

**Review by FlyingRatMan — APPROVED** [review]

**Pull request merged**